### DISCUSSION TITLE
Option to shorten Nvim-tree path?

### DISCUSSION THREAD

**Discussion by rodhash** [nvim-lualine/lualine.nvim#684] [Configuration]
Hi,

Do we have an option to not show full path when we open nvim-tree? I have the extension enable as it shows the full-path in the bottom:

https://user-images.githubusercontent.com/29671981/169314227-a10f0b3c-fbc4-4c03-9a05-8afc3d679422.mp4

Thanks

**Comment by shadmansaleh** [comment]
You can use a [custom extension](https://github.com/nvim-lualine/lualine.nvim#custom-extensions) there you can set appropriate options for filename component or have a custom component that does what you want
